ocs_ci/ocs/ocs_upgrade.py

```diff
@@ -110,12 +110,9 @@ def verify_image_versions(old_images, upgrade_version):
     )
     verify_pods_upgraded(old_images, selector=constants.MON_APP_LABEL, count=3)
     verify_pods_upgraded(old_images, selector=constants.MGR_APP_LABEL)
-    # OSD upgrade have timeout 10mins for new attempt if cluster is not health.
-    # https://bugzilla.redhat.com/show_bug.cgi?id=1840729 setting timeout for
-    # 12.5 minutes per OSD
     verify_pods_upgraded(
         old_images, selector=constants.OSD_APP_LABEL, count=osd_count,
-        timeout=750 * osd_count,
+        timeout=600 * osd_count,
```

Comment:
what about upgrade from 4.3 to 4.4.  If we need to still run that, then we can maybe have an issue and maybe we should change this value only when version >= 4.5

Reply:
i can add it as constant and use it according to upgraded version

Reply:
Please do @AviadP